config/samples/kustomization.yaml

```diff
@@ -1,4 +1,8 @@
 ## Append samples of your project ##
+namespace: h3poteto
```

Comment:
we are going to leave this namespace for test ?

Reply:
Oh, right. Which namespace is better? Should I create `test` namespace?

Reply:
any name is fine, maybe "rollout-test"?

Reply:
OK, I updated

Reply:
Thank you